test/integration/spec/connect.js

```diff
@@ -1271,6 +1271,59 @@ describe('connect', function() {
         });
       });
     });
+
+    if (defaults.topology !== 'peer-to-peer') {
```

Comment:
@makarandp0 This test should be in `test/integration/spec/room.js`. Also, can you test whether the event is fired on `RemoteTrackPublication` and `RemoteParticipant` as well?

Reply:
I am already verifying that event fires on track/RemoteTrackPublication/RemoteParticipant and room. But sure, I will move it to room.js,